static/fonts.css

```diff
@@ -0,0 +1,132 @@
+/******** FONTS *********/
+
+@font-face {
+	font-family: "Mercury";
+	src: url('fonts/mercury/regular/mercury.eot');
+	src: url('fonts/mercury/regular/mercury.eot?#iefix') format('embedded-opentype'), 
+		 url('fonts/mercury/regular/mercury.woff') format('woff'),
+		 url('fonts/mercury/regular/mercury.ttf') format('truetype'),
+		 url('fonts/mercury/regular/mercury.svg#Mercury') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: "MercuryBold";
+	src: url('fonts/mercury/bold/mercury-bold.eot');
+	src: url('fonts/mercury/bold/mercury-bold.eot?#iefix') format('embedded-opentype'),
+	     url('fonts/mercury/bold/mercury-bold.woff') format('woff'),
+	     url('fonts/mercury/bold/mercury-bold.ttf') format('truetype'),
+	     url('fonts/mercury/bold/mercury-bold.svg#MercuryBold') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: "CTBoomerCon";
```

Comment:
Do we use this font?

Reply:
this is an old file we use in a bunch of projects; CTBoomer is the font we use for a lot of print graphics. we don't use it in this project so i took it out.